x-pack/plugins/maps/public/classes/styles/vector/style_meta.ts

```diff
@@ -14,13 +14,15 @@ export class StyleMeta {
   }
 
   getRangeFieldMetaDescriptor(fieldName: string): RangeFieldMeta | null {
-    return this._descriptor.fieldMeta[fieldName] && this._descriptor.fieldMeta[fieldName].range
+    return this._descriptor.fieldMeta[fieldName]
```

Comment:
Check for this._descriptor.fieldMeta[fieldName].range is still needed since fieldMeta[fieldName].range is defined as `?:`